src/main/java/com/LuuQu/medicalclinic/controller/PatientController.java

```diff
@@ -0,0 +1,47 @@
+package com.LuuQu.medicalclinic.controller;
+
+import com.LuuQu.medicalclinic.model.dto.PatientDto;
+import com.LuuQu.medicalclinic.service.PatientService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/patients")
+public class PatientController {
+    private final PatientService patientService;
+
+    @GetMapping
+    public List<PatientDto> getPatientList() {
+        return patientService.getPatients();
+    }
+
+    @GetMapping("/{email}")
+    public PatientDto getPatient(@PathVariable String email) {
+        return patientService.getPatient(email);
+    }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public PatientDto addPatient(@RequestBody PatientDto patient) {
```

Comment:
bardziej pytanie niż sugestia: dlaczego w @RequestBody jest typ PatientDto a nie Patient?

Reply:
Teraz to nie robi większej różnicy, ale zakładając, że pacjent w bazie danych jest zapisany w kilku tabelach np. 
Osoba (Id, Imię, Nazwisko), User(Id, Login, Password), Patient(reszta)
To użytkownik nadal wysyła 1 cały model z danymi, gdzie w serwisie zapisuję dane do kilku tabel, czyli przez kilka entity